VERSION

```diff
@@ -1 +1 @@
-0.25.1
+0.26.0
```

Comment:
Just curious for my understanding, why did you update the minor version number instead of only the patch version?

Reply:
because it's new functionality rather than a fix